apps/aecore/src/aec_upnp.erl

```diff
@@ -87,6 +118,17 @@ code_change(_OldVsn, State, _Extra) ->
 %%% Internal functions
 %%%===================================================================
 
+kill_mapping_worker(#state{pid = undefined, mon = undefined}) ->
+    ok;
+kill_mapping_worker(#state{pid = Pid, mon = Ref}) ->
+    demonitor(Ref, [flush]),
+    true = exit(Pid, shutdown),
```

Comment:
I do not see the spawned process as trapping exits (I did not check the internals of the nat library though) so I understand this is intended to kill the process. (`true =` always succeeds, even if the `Pid` is not there any longer). Ok.